Makefile

```diff
@@ -122,7 +125,8 @@ build: generate fmt vet ## Build manager binary.
 
 .PHONY: run
 run: manifests generate fmt vet ## Run a controller from your host.
-	go run ./main.go
+	@echo "warn: webhook support is disabled. In case you rely on some kind of webhook, please use another deployment method!"
+	ENABLE_WEBHOOKS=$(ENABLE_WEBHOOKS) go run ./main.go
```

Comment:
I don't think we need to make `ENABLE_WEBHOOKS` configurable in the makefile:
* the run target fails if webhook is enabled
* `ENABLE_WEBHOOKS ?= false` looks weird in the makefile, it makes impression that the webhooks are disabled in the normal build as well. The opposite is true.

Reply:
Sure we can also get rid of it. Its in because i followed the opentelemetry-operator approach.